test/stablex/stablecoin_converter.js

```diff
@@ -46,6 +46,7 @@ contract("StablecoinConverter", async (accounts) => {
   let BATCH_TIME
   before(async () => {
     const feeToken = await MockContract.new()
+    await feeToken.givenAnyReturnBool(true)
```

Comment:
Sorry about that should be addressed by https://github.com/gnosis/mock-contract/pull/17

Reply:
Very nice! I will wait till its merged and try it out.